nautobot/ipam/api/serializers.py

```diff
@@ -218,7 +218,7 @@ class VLANSerializer(NautobotModelSerializer, TaggedModelSerializerMixin, Status
     url = serializers.HyperlinkedIdentityField(view_name="ipam-api:vlan-detail")
     site = NestedSiteSerializer(required=False, allow_null=True)
     location = NestedLocationSerializer(required=False, allow_null=True)
-    group = NestedVLANGroupSerializer(required=False, allow_null=True)
+    vlan_group = NestedVLANGroupSerializer(required=False, allow_null=True)
```

Comment:
This needs to be documented in the upgrading documentation as a renamed serializer field.